pyam/utils.py

```diff
@@ -116,6 +116,11 @@ def read_pandas(path, default_sheet='data', *args, **kwargs):
         if len(xl.sheet_names) > 1 and 'sheet_name' not in kwargs:
             kwargs['sheet_name'] = default_sheet
         df = pd.read_excel(path, *args, **kwargs)
+
```

Comment:
Is this change related to the PR? Could not figure out why.

Reply:
Yes, this is not related to the PR per se, but fixes a pandas v1.2 issue on the fly.

I started working on a nightly test suite so that we don't catch these incompatibilities while working on new features, but can resolve them in mini-PRs when they arise...